styles.css

```diff
@@ -1 +1,120 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container {
+    display: grid;
+    color: white;
+    padding: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+        "a"
+        "b"
+        "c"
+        "d"
+        "e"
+        "."
+        "f";
+    row-gap: 16px;
```

Comment:
Nova had hier ook een column-gap staan, maar bij de mobile versie is er geen tweede column dus zou dat in principe niet nodig zijn.